docs/en/sql-reference/statements/alter.md

```diff
@@ -197,6 +197,16 @@ Constraint check *will not be executed* on existing data if it was added.
 
 All changes on replicated tables are broadcasting to ZooKeeper so will be applied on other replicas.
 
+### Manipulations with Replicas {#manipulations-with-replicas}
+
+Replicas can be dropped using following syntax:
+
+```sql
+ALTER TABLE [db].name DROP REPLICA replica_name;
+```
+
+Queries will remove the replica path in zookeeper, it's useful when you want to decrease your replica factor. It will only drop the inactive replica, and it can't drop local replica, please use `DROP TABLE`.
```

Comment:
What happens with data on that replica? It remains on remote server?

Reply:
Yes, it remains. We can't control that.  The main reason we call `DROP REPLICA` is that we should enable remove the 'dead' replica.